project/LookupTableGen.scala

```diff
@@ -0,0 +1,225 @@
+package org.typelevel.idna4s.build
+
+import cats._
+import cats.data._
+import cats.syntax.all._
+import scala.annotation.tailrec
+import scala.collection.immutable.SortedSet
+import scala.collection.immutable.SortedMap
+
+object LookupTableGen {
+
+  // Why is this not in cats?
+  private implicit def orderingFromOrder[A](implicit A: Order[A]): Ordering[A] =
+    A.toOrdering
```

Comment:
Possibly creates a cycle if we have an implicit going the other direction? 🤔

Reply:
I suspect you're correct.